README.md

```diff
@@ -78,3 +78,29 @@ Follow the commands to make the development process easier:
 5. This API returns the list of all URLs/Routes that are being monitored for testing using the application.
 
 For more information, regarding usage in different languages. Visit [Bench-Routes](https://documenter.getpostman.com/view/6521254/SzRuWqq9?version=latest).
+
+## GSSoC'20   
+**Bench-routes** has been selected at [GirlScript Summer of Code 2020](https://www.gssoc.tech/)!!! -a three months long Open Source program conducted by [GirlScript Foundation](https://www.girlscript.tech/home).
+
+### 👨 Project Owner
+
+- Harkishen Singh <p>[<img src="https://img.icons8.com/windows/32/000000/github-2.png" display = "inline-block">](https://github.com/Harkishen-Singh) [<img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2.png"/>](https://www.linkedin.com/in/harkishen-singh/)</p>
+
+### 👬  Mentors
+
+- Aquib Baig <p>[<img src="https://img.icons8.com/windows/32/000000/github-2.png" display = "inline-block">](https://github.com/aquibbaig) [<img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2.png"/>](https://www.linkedin.com/in/baigaquib/)</p>
+
+- Ganesh Patro <p>[<img src="https://img.icons8.com/windows/32/000000/github-2.png" display = "inline-block">](https://github.com/ganeshpatro321) [<img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2.png"/>](https://www.linkedin.com/in/ganeshpatro321/)</p>
+
+- Marut Pandya <p>[<img src="https://img.icons8.com/windows/32/000000/github-2.png" display = "inline-block">](https://github.com/pandyamarut) [<img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2.png"/>](https://www.linkedin.com/in/marut-pandya-a16480131/)</p>
+
+- Vatsal Trivedi <p>[<img src="https://img.icons8.com/windows/32/000000/github-2.png" display = "inline-block">](https://github.com/vattytrivedi) [<img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2.png"/>](https://www.linkedin.com/in/trivedi-vatsal/)</p>
+
+Feel free to ask your queries!! 🙌
+
+### Slack Channel
+
+- [#proj_bench-routes](https://app.slack.com/client/TRN1H1V43/CUCKG6R0V)
+
```

Comment:
Let the name of mentors have email-id only. Also, add these mentors after Ganesh Patro:

```
Muskan Khedia (muskan.khedia2000@gmail.com)
Ankit Jena (ankitjena13@gmail.com)
```
Email IDs are enough.

Also,, instead of slack channel, use the **gitter channel** since slack channel is for gssoc only. We need a universal channel for everyone to discuss.